go.mod

```diff
@@ -68,7 +68,10 @@ require (
 	github.com/gorilla/websocket v1.4.2
 	github.com/gosnmp/gosnmp v1.33.0
 	github.com/grid-x/modbus v0.0.0-20210224155242-c4a3d042e99b
-	github.com/gwos/tcg/sdk v0.0.0-20211130162655-32ad77586ccf
+	github.com/grid-x/serial v0.0.0-20191104121038-e24bc9bf6f08 // indirect
```

Comment:
Can you run `go mod tidy`? These indirect dependencies should be moved to the second require block, not sure why it isn't being enforced by Go yet.

Reply:
done